webapp/channels/src/components/admin_console/access_control/policy_details/policy_details.tsx

```diff
@@ -0,0 +1,426 @@
+                            expanded={true}
+                            className={'console delete-policy'}
+                        >
+                            <Card.Header>
+                                <TitleAndButtonCardHeader
+                                    title={
+                                        <FormattedMessage
+                                            id='admin.access_control.policy.edit_policy.delete_policy.title'
+                                            defaultMessage='Delete policy'
+                                        />
+                                    }
+                                    subtitle={
+                                        hasChannels() ? (
+                                            <FormattedMessage
+                                                id='admin.access_control.policy.edit_policy.delete_policy.subtitle.has_resources'
+                                                defaultMessage='Remove all assigned resources (eg. Channels) to be able to delete this policy'
+                                            />
+                                        ) : (
+                                            <FormattedMessage
+                                                id='admin.access_control.policy.edit_policy.delete_policy.subtitle'
+                                                defaultMessage='This policy will be deleted and cannot be recovered.'
+                                            />
+                                        )
+                                    }
+                                    buttonText={
+                                        <FormattedMessage
+                                            id='admin.access_control.policy.edit_policy.delete_policy.delete'
+                                            defaultMessage='Delete'
+                                        />
+                                    }
+                                    onClick={() => {
+                                        if (hasChannels()) {
+                                            return;
+                                        }
+                                        handleDelete();
+                                    }}
+                                    isDisabled={hasChannels()}
+                                />
+                            </Card.Header>
+                        </Card>
+                    )}
+                </div>
+            </div>
+
+            {addChannelOpen && (
+                <ChannelSelectorModal
+                    onModalDismissed={() => setAddChannelOpen(false)}
+                    onChannelsSelected={(channels) => handleChannelChanges(channels, true)}
+                    groupID={''}
+                    alreadySelected={Object.values(channelChanges.added).map((channel) => channel.id)}
+                    excludeAccessControlPolicyEnforced={true}
+                    excludeTypes={['O', 'D', 'G']}
```

Comment:
@isacikgoz do we also need to exclude those channels that have the groups sync enabled here? We discussed that if a channel is synched with LDAP groups, they won't be able to add a policy unless they disable that first - in the UI we discussed that in the channels specific page but here we probably need to filter as well https://hub.mattermost.com/pde/pl/iget5qwagf8nzea1metoz66c5y

Reply:
Yep, that is actually excluded in the backend. Even to be more strict we are narrowing down to private channels as well (backend).